packages/block-editor/src/components/block-tools/index.js

```diff
@@ -106,13 +94,15 @@ export default function BlockTools( {
 			) {
 				return;
 			}
-			const clientIds = getSelectedBlockClientIds();
-			if ( clientIds.length > 1 ) {
-				event.preventDefault();
-				removeBlocks( clientIds );
+
+			if ( clientIds.length < 2 ) {
```

Comment:
Some of these cases could use a comment to describe the situation we are addressing